crates/fuel-core/src/state/historical_rocksdb.rs

```diff
@@ -270,16 +287,30 @@ where
         Ok(())
     }
 
+    fn remove_v1_entries(&self) -> StorageResult<()> {
+        if !self.has_v1_history() {
+            return Ok(())
+        }
+
+        self.has_v1_history
+            .store(false, core::sync::atomic::Ordering::Release);
+        self.db
+            .clear_table(ModificationsHistoryV1::<Description>::column())?;
```

Comment:
Shall we flip these two calls to only store `false` when the `clear_table()` actually finishes w/o error?

Reply:
sounds good, feel free to push that fix up :)

Reply:
addressed in d02ec975cd